src/edu/csus/ecs/pc2/core/standings/ScoreboardUtilites.java

```diff
@@ -188,6 +190,95 @@ public static ScoreboardJsonModel createContestStandingsFromJSON(File jsonFile)
     public static int toInt(String string, int defaultNumber) {
         return Utilities.nullSafeToInt(string, defaultNumber);
     }
-    
-    
+
+    /**
+     * 
+     * @param clientId
+     * @param contest
+     * @return
+     */
+    public static Run[] getRunsForUserDivision(ClientId clientId, IInternalContest contest) {
+
+        String division = getDivision(contest, clientId);
+        
+//        System.out.println("debug 22 getRunsForUserDivision for "+clientId+" div is "+division);
+
+        if (ClientType.Type.TEAM.equals(clientId.getClientType())) {
+
+            List<Run> teamRuns = new ArrayList<Run>();
```

Comment:
`teamRuns` is a very misleading name for this variable -- the variable in fact is holding not just the runs for the specified team, but also all runs for other teams in the same division.  It seems like "divisionRuns" would be a better name.

Reply:
will rename, but they are teams' runs.

Reply:
pushed